src/main/java/io/vertx/cassandra/CassandraClient.java

```diff
@@ -30,6 +30,9 @@
 
 /**
  * Eclipse Vert.x Cassandra client.
+ *
+ * @author Pavel Drankou
+ * @author Thomas Segismont
```

Comment:
It's not necessary to add me :) Just the original author

Reply:
I think is fine to have you as an author here, because your reviews had a big influence on the client